packages/ds-medicare-gov/src/styles/components/_SimpleFooter.scss

```diff
@@ -59,16 +59,9 @@
     gap: 0 $spacer-2;
   }
 
-  @media screen and (min-width: $media-width-sm) {
-    flex-direction: column;
-  }
-
   @media screen and (min-width: $media-width-lg) {
-    justify-content: space-between;
-  }
-
-  @media screen and (min-width: $media-width-xl) {
```

Comment:
removing the glossary link gives more space to that line of links. looks better when merging lg and xl breakpoint styles and calling it a day